app/src/main/java/org/fossasia/openevent/general/event/EventDetailsFragment.kt

```diff
@@ -177,11 +177,13 @@ class EventDetailsFragment : Fragment() {
         if (!event.description.isNullOrEmpty()) {
             setTextField(rootView.eventDescription, event.description?.stripHtml())
 
-            if (rootView.eventDescription.lineCount > LINE_COUNT) {
-                rootView.seeMore.visibility = View.VISIBLE
-                // start about fragment
-                rootView.eventDescription.setOnClickListener(aboutEventOnClickListener)
-                rootView.seeMore.setOnClickListener(aboutEventOnClickListener)
+            rootView.eventDescription.post {
```

Comment:
Why do we need post?

Reply:
so the reason why "see more" disappear is because rootView.eventDescription.lineCount is equal 0. From what I understand and research on StackOverflow, the method lineCount() is called only the view is rendered. I use post() method to check for lineCount after the view is rendered. Correct me if I am wrong in this PR. Thanks